src/Attachment.tsx

```diff
@@ -170,34 +169,44 @@ export const AttachmentView = (props: {
         case "application/vnd.microsoft.card.hero":
             if (!attachment.content)
                 return null;
+
+            const heroCardBuilder = new CardBuilder.AdaptiveCardBuilder();
```

Comment:
We should create helper methods to create the various card types for readability.